### PR TITLE
Bug 1213381 - [Music][NGA] Remove A-Z picker until we have a UX spec …

### PR DESCRIPTION
**Pull request by justindarc**
…for additional locales

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=cf4fb8770dc0dc9c5ea3e54ec1927aed679fbb0a

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4484132c7606f42c4450fe609e6f1414c68ef746

File: apps/music/views/albums/view.js [hunk 1]
```diff
@@ -64,31 +60,19 @@ AlbumsView.prototype.render = function() {
 };
 
 AlbumsView.prototype.getAlbums = function() {
-  return document.l10n.formatValues('unknownAlbum', 'unknown')
-    .then(([unknownAlbum, unknown]) => {
+  return document.l10n.formatValues('unknownAlbum', 'unknownArtist')
```
Comment (jimporter) [inline apps/music/views/albums/view.js]: I'm not sure I understand why unknownArtist was added in this patch. Is that just another bug that you're fixing at the same time here?

File: apps/music/views/songs/view.js [hunk 2]
```diff
@@ -71,24 +67,18 @@ SongsView.prototype.render = function() {
 
 SongsView.prototype.getSongs = function() {
   console.time('getSongs');
-  return document.l10n.formatValue('unknown')
-    .then((unknown) => {
+  return document.l10n.formatValues('unknownTitle', 'unknownArtist')
```
Comment (jimporter) [inline apps/music/views/songs/view.js]: Ditto here about the unknownArtist string.

File: apps/music/views/albums/view.js (hunk 1 above)
Comment (justindarc) [inline apps/music/views/albums/view.js]: see below on line 73.. we show the `artist` as the sub-text in the list items and we weren't falling back to the 'unknownArtist' string if that field was blank

File: apps/music/views/songs/view.js (hunk 2 above)
Comment (justindarc) [inline apps/music/views/songs/view.js]: Same reasons as above.

**Pull request merged**